airbyte-integrations/connectors/source-commcare/Dockerfile

```diff
@@ -35,5 +35,5 @@ COPY source_commcare ./source_commcare
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.1.1
```

Comment:
The version also needs to be changed in `metadata.yaml`.

Reply:
changed it